xds/src/main/java/io/grpc/xds/XdsLbState.java

```diff
@@ -85,6 +105,35 @@ final XdsComms shutdownAndReleaseXdsComms() {
     return xdsComms;
   }
 
+  /**
+   * Manages EAG and locality info for a collection of subchannels, not including subchannels
+   * created by the fallback balancer.
+   */
+  static final class SubchannelStore {
```

Comment:
Is this essentially the round-robin list, because I see "drop" is indicated here?
If so, it should probably be named more specific, say, "RoundRobinList".

Reply:
Uncertain yet. It may have hierarchies, because it may be grouped by locality. Will see how later implementation goes. Also, it may be weighted RR over localities then intra-locality.